### DISCUSSION TITLE
[本地部署] SSL wrong version number应该怎么办

### DISCUSSION THREAD

**Discussion by Justinyxj** [GaiZhenbiao/ChuanhuChatGPT#638] [General]
[本地部署] SSL wrong version nmber应该怎么办

执行程序如下，已更新urllib及request库到指定版本，提示错误代码见下图

### 错误截图
![image](https://user-images.githubusercontent.com/111212705/227134286-13ec8da3-f918-4d3e-b895-7e4d3ec926bd.png)

请问怎么调整ssl版本，应该对应什么版本号

#### 桌面系统
 - OS: [Windows10]

**Comment by qlhazy** [comment]
同问

**Comment by liuyuanjun520** [comment]
同问

**Comment by MZhao-ouo** [comment]
奇怪的问题，请移至Discussing

**Comment by xmlv** [comment]
urllib3的版本可能高了，要干掉装个低版本的。 翻一下疑难杂症文档，有说这个问题。
